### PR TITLE
LITE-28162 fix export/sync/clone for unit of measure

### PR DESCRIPTION
**Pull request by ffaraone**

### CODE REVIEW

**Comment by vgrebenschikov** [comment]
what happened on import?

File: connect/cli/plugins/product/sync/items.py [hunk 1]
```diff
@@ -274,6 +282,12 @@ def _get_or_create_unit(self, data):
             if unit['type'] == data.type and unit['description'] == data.unit:
                 return unit['id']
 
+        for unit in self._shared_units:
+            if unit['id'] == data.unit:
+                return unit['id']
+            if unit['type'] == data.type and unit['description'] == data.unit:
```
Comment (bdjilka) [inline connect/cli/plugins/product/sync/items.py:288]: In export name of unit is written, so it is more clear if data will be compared by name, not description. WDYT?

File: connect/cli/plugins/product/sync/items.py [hunk 2]
```diff
@@ -306,6 +320,7 @@ def _get_item_payload(self, data):
             'period': self._get_billing_period(data),
             'ui': {'visibility': True},
         }
+        print(payload)
```
Comment (bdjilka) [inline connect/cli/plugins/product/sync/items.py:323]: print

**Review by bdjilka — CHANGES_REQUESTED** [review]

File: connect/cli/plugins/product/sync/items.py (hunk 1 above)
Comment (ffaraone) [reply connect/cli/plugins/product/sync/items.py:288]: the unit collection return the following representation:

```
    {
        "id": "CU-876-577",
        "unit": "unit",
        "description": "Boolean",
        "owner": {
            "id": "VA-037-000",
            "name": "Vendor account 00 for ffaraone"
        },
        "type": "reservation",
        "cumulative": false
    },
    {
        "id": "boolean",
        "unit": "unit",
        "description": "Boolean",
        "type": "reservation",
        "cumulative": false
    }
```

There is no name attribute there. But the unit representation inside the product item representation call it `name`.

File: connect/cli/plugins/product/sync/items.py (hunk 2 above)
Comment (ffaraone) [reply connect/cli/plugins/product/sync/items.py:323]: Thanks!

File: connect/cli/plugins/product/sync/items.py (hunk 1 above)
Comment (bdjilka) [reply connect/cli/plugins/product/sync/items.py:288]: Oh, I understood

**Review by bdjilka — APPROVED** [review]

**Pull request merged**